PYME/DSView/DisplayOptionsPanel.py

```diff
@@ -285,15 +285,15 @@ def OnIsodataThresh(self, event=None, bin_spacing='linear'):
         
         
         for i, hClim in enumerate(self.hcs):
-            t = thresholding.isodata_f(self.do.ds[:,:,:,i], bin_spacing=bin_spacing)
+            t = thresholding.isodata_f(self.do.ds[:,:,:,:,i], bin_spacing=bin_spacing)
```

Comment:
Probably OK for now, but this might need a slightly more complicated solution, as I'm not sure that `do.ds` is strictly guaranteed to be xyztc.